exercise_02/createUser.js

```diff
@@ -13,3 +13,23 @@
  * output
  *  - string
  */
+/**
+ * It will generate a random number in between 0 and 1
+   If random number is less than 0.5 or email is empty, then call onError callback
+   Otherwise, call onSuccess function
+ * @param {string} email
+ * @param {Function} onError
+ * @param {Function} onSuccess
+ * @returns string
+ */
+
+const createUser = function (email, onError, onSuccess) {
```

Comment:
Take a look at comment in exercice `transformStringToArray.js`

FIX: apply validation and error handling as first step